Automattic/jetpack
# AI Chat: Fix feedback styles and icon SVGs

## Proposed changes:


This PR fixes the style of the feedback section: 
- Uses theme styles for button colors 
- Matches styles for the components used in the block and elsewhere in WP

This PR also adds a few SVG's for use in the shared icons. This means that icons will now render on the front end of the site when not logged in. The alternate to this was to load all dashicons styles on the front end, which did not seem ideal. 

The SVGs were copied from seth's site, so they will match the styling there.  

<img width="772" alt="image" src="https://github.com/Automattic/jetpack/assets/7129409/39586950-1de8-4b47-ac71-e2e42d83f20a">

## Jetpack product discussion


N/A

## Does this pull request change what data or activity we track or use?


No

## Testing instructions:






- Ask a question 
- Once answered, you should see the feedback buttons in the same color as your theme. 
- Click the copy button, make sure it looks ok
- Click one of the thumbs. The input field should display nicely. 

To achieve maximum testing, do those steps while logged out incognito or something.
